sdk/iot/core/inc/az_iot_hub_client.h

```diff
@@ -0,0 +1,505 @@
+    az_iot_hub_client const* client,
+    az_span mqtt_topic_filter,
+    az_span* out_mqtt_topic_filter);
+
+/**
+ * @brief Gets the MQTT topic filter to subscribe to twin desired property changes.
+ * @note The payload will contain only changes made to the desired properties.
+ *
+ * @param client
+ * @param mqtt_topic_filter An empty #az_span with sufficient capacity to hold the MQTT topic
+ *                          filter.
+ * @param out_mqtt_topic_filter The output #az_span containing the MQTT topic filter.
+ * @return #az_result
+ */
+AZ_NODISCARD az_result az_iot_hub_client_twin_patch_subscribe_topic_filter_get(
+    az_iot_hub_client const* client,
+    az_span mqtt_topic_filter,
+    az_span* out_mqtt_topic_filter);
+
+/**
+ * @brief Twin response type.
+ *
+ */
+typedef enum
+{
+  AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_GET = 1,
+  AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES = 2,
+  AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES = 3,
+} az_iot_hub_client_twin_response_type;
+
+/**
+ * @brief Twin response.
+ *
+ */
+typedef struct az_iot_hub_client_twin_response
+{
+  az_iot_hub_client_twin_response_type response_type; /** @brief Twin response type. */
+  az_iot_hub_client_status status; /** @brief The operation status. */
+  az_span request_id; /** @brief Request ID matches the ID specified when issuing a Get or Patch
+                         command. */
+  az_span version; /** @brief The Twin object version.
+                    * @note This is only returned when
+                    * response_type==AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_DESIRED_PROPERTIES or
+                    * response_type==AZ_IOT_CLIENT_TWIN_RESPONSE_TYPE_REPORTED_PROPERTIES */
+} az_iot_hub_client_twin_response;
+
+/**
+ * @brief Attempts to parse a received message's topic.
+ *
+ * @param client
+ * @param received_topic An #az_span containing the received topic.
+ * @param out_request If the message is twin-operation related, this will contain the
+ *                    #az_iot_hub_client_twin_response.
+ * @return #az_result
+ */
+AZ_NODISCARD az_result az_iot_hub_client_twin_received_topic_parse(
+    az_iot_hub_client const* client,
+    az_span received_topic,
+    az_iot_hub_client_twin_response* out_twin_response);
+
+/**
+ * @brief Gets the MQTT topic that must be used to submit a Twin GET request.
+ * @note The payload of the MQTT publish message should be empty.
+ *
+ * @param client
+ * @param request_id The request id.
+ * @param mqtt_topic An empty #az_span with sufficient capacity to hold the MQTT topic.
+ * @param out_mqtt_topic The output #az_span containing the MQTT topic.
+ * @return #az_result
+ */
+AZ_NODISCARD az_result az_iot_hub_client_twin_get_publish_topic_get(
+    az_iot_hub_client const* client,
+    az_span request_id,
+    az_span mqtt_topic,
+    az_span* out_mqtt_topic);
+
+/**
+ * @brief Gets the MQTT topic that must be used to submit a Twin GET request.

```

Comment:
>Gets the MQTT topic that must be used to submit a Twin GET request. [](start = 10, length = 67)

Comment is a copy paste of line 468 for az_iot_hub_client_twin_get_publish_topic_get?